src/Api/AdminConsole/Controllers/PoliciesController.cs

```diff
@@ -208,8 +209,34 @@ public async Task<PolicyResponseModel> Put(Guid orgId, PolicyType type, [FromBod
             throw new BadRequestException("Mismatched policy type");
         }
 
+
+
         var policyUpdate = await model.ToPolicyUpdateAsync(orgId, _currentContext);
+
+
         var policy = await _savePolicyCommand.SaveAsync(policyUpdate);
+
+        var savePolicyRequest = GetMockedSavePolicyRequest();
+
+        var mockReponse = await _savePolicyCommand.SavePrototypeAsync(savePolicyRequest);
+
         return new PolicyResponseModel(policy);
     }
+
+    private static SavePolicyRequest GetMockedSavePolicyRequest()
+    {
+        var savePolicyRequest = new SavePolicyRequest(
+            new PolicyUpdate
+            {
+                OrganizationId = Guid.Parse("aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa"),
+                Type = PolicyType.OrganizationDataOwnership,
+                Enabled = true,
+                Data = ""
+            },
+            new OrganizationModelOwnershipPolicyModel("Default Collection Name", executeSideEffect: true),
```

Comment:
@eliykat 
So at the controller level, we can detect the behavior of the validator. 

Of course, this is just a mock, so we should move this logic to where it makes more sense. The endpoint doesn’t need to know about this.